resources/i18n/en.json

```diff
@@ -1066,6 +1066,7 @@
     "CancelLogin": "Cancel Login",
     "SignoutFinished": "Signout finished.",
     "APIEndpointEmpty": "API Endpoint is empty. Please specify API endpoint to login.",
+    "APIEndpointStartWith": "API Endpoint must start with http:// or https://.",
```

Comment:
Please use i18n.
1. Run `make i18n`
2. Translate other words if you can.

More details: https://github.com/lablup/backend.ai-webui#extracting-i18n-resources